### PR TITLE
task(upm): prepare for initial upm release

### PR DESCRIPTION
**Pull request by rich-bugsnag**
## Goal

Get UPM tools and repo ready for release

## Changeset

- Updated contributing release instructions
- Updated upm repo readme file
- Added asembly definition files to auto build script

## Testing

Manually tested

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -133,16 +133,21 @@ bundle exec maze-runner features/handled_errors.feature
...
-3. merge the latest `master` into `upm-latest` 
+3. Test that the built package installs by using the install local package option in unity package manager.
 
-4. Build the upm package by running the `build-upm-package.sh` script in the upm-tools directory. You should pass the version number of the release like so `./build-upm-package.sh 6.x.x`. You must run the script from within the upm-tools folder.
-5. Test that the built package installs by using the install local package option in unity package manager.
-6. Deploy the package with the `deploy-upm.sh` script passing the release version like so `./deploy-upm.sh 6.x.x`. This will commit the package and tag the upm release
+4. Clone the `bugsnag-unity-upm` repo and make sure you are in the `main` branch.
 
+5. Replace the contents of the repo with the contents of the `upm-package` directory in the `bugsnag-unity` repo
+6. Commit these changes to main with the message `Bugsnag Unity UPM release V6.x.x`
```
Comment (tomlongridge) [inline CONTRIBUTING.md:145]: nit: we have somewhat of a convention elsewhere for just 'Release vx.x.x' - repeating the repo name doesn't really add anything.

File: upm-tools/AssemblyDefinitions/Bugsnag.asmdef [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿{
```
Comment (tomlongridge) [inline upm-tools/AssemblyDefinitions/Bugsnag.asmdef:1]: Should the AssemblyDefinitions be checked-in or are they generated?

Comment (rich-bugsnag) [reply upm-tools/AssemblyDefinitions/Bugsnag.asmdef:1]: they should be checked in

File: CONTRIBUTING.md (hunk 1 above)
Comment (rich-bugsnag) [reply CONTRIBUTING.md]: done 👍

**Review by tomlongridge — APPROVED** [review]

**Pull request merged**